components/dashboard/src/Login.tsx

```diff
@@ -26,14 +26,6 @@ export function markLoggedIn() {
     document.cookie = GitpodCookie.generateCookie(window.location.hostname);
 }
 
-export function hasLoggedInBefore() {
-    return GitpodCookie.isPresent(document.cookie);
```

Comment:
**question**: GitpodCookie seems to be used elsewhere, :a: do we still need it and :b: is there any other place we need to update for this change? Cc @geropl & @AlexTugarev because https://github.com/gitpod-io/gitpod/pull/6271.

Reply:
The build was failing because this function is also used in [`QueryErrorBoundary.tsx`](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/src/components/error-boundaries/QueryErrorBoundary.tsx). Added back and rebased.